### ISSUE TITLE
Cart Module Not working it freezes when clicked on: sharpedge CMS V 3.42.76 

### ISSUE THREAD

**Issue by ebonylegg** [srpurdy/SharpEdgeCMS#39]
There appears to be a problem with the shopping cart. when the cart icon is clicked, it just freezes and nothing happens. Sometimes a status bar on top right shows up but it keeps displaying in an indefinite loop and nothing happens. and yes, the paypal is active and set with an active paypal account.

**Comment by srpurdy** [comment]
This looks like a bug. I'll look into it

**Comment by srpurdy** [comment]
http://sedemo.sharpedgecms.com/en/products/category/Test

I have fixed this bug, I will have an update on github shortly. The CMS will also have an update. So it can be updated using the updater also. (It will only update the default theme) So if you already have a custom theme you will want to copy the file from /sharpedge/views/themes/default_bootstrap/widget_views/cart_widget_view.php. Only copy this file once I have updated the repo here on github. I will post here once it is up.
